packages/runner-ct/src/lib/event-manager.js

```diff
@@ -195,6 +196,17 @@ const eventManager = {
       this._clearAllCookies()
     })
 
+    const hmrSocket = connectWebpackHmr({
+      url: `${window.location.origin}/cypress-webpack-hmr-socket`,
+      onReload: () => {
+        if (!state.isInitialBuildSucceed) {
```

Comment:
not a blocker, but what do you think about putting the positive statement first in the if statement? eg:

```
if (state.initialBuildSucceed) {
  rerun() // most likely path
} else {
  // ... not often happening
}
```

just a style thing, no need to make this change to get the PR merged :)